conf-sample.edn

```diff
@@ -17,11 +17,12 @@
  :resources {:group-id "com.elasticpath.rest.definitions"
              :format "%s/ep-resource-%s-api"
              :version "0-SNAPSHOT"
-             :modules #{addresses
-                        assets
+             :modules #{
```

Comment:
small nitpick, could you please remove the empty lines there? Aka data in contained inside the brackets. It reads more idiomatic :smile: